### PR TITLE
feat: Code fixes

### PR DESCRIPTION
**Pull request by karannakra**
## **📝 Changes Description 📝**

✅ Use clsx package

### CODE REVIEW

**Comment by JLarky** [comment]
> otherwise eslint will throw error

with what config?

**Comment by karannakra** [comment]
> > otherwise eslint will throw error
> 
> 
> 
> with what config?

@typescript-eslint/restrict-plus-operand

https://typescript-eslint.io/rules/restrict-plus-operands/#:~:text=Require%20both%20operands%20of%20addition,bigint%20%2C%20number%20%2C%20or%20string%20.&text=Extending%20%22plugin%3A%40typescript%2Deslint,ESLint%20configuration%20enables%20this%20rule.&text=This%20rule%20requires%20type%20information%20to%20run.

**Comment by karannakra** [comment]
> > otherwise eslint will throw error
> 
> with what config?

image

File: src/components/app/Settings.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { useQueryClient } from '@tanstack/react-query';
 import { getQueryKey } from '@trpc/react-query';
+import classNames from 'clsx';
```
Comment (JLarky) [inline src/components/app/Settings.tsx:3]: let's use `clsx` like in the rest of the app

**Pull request merged**